src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
Having all the things injected as globals, is like loading jquery via script tag and using it all over the place without importing it explicitly. I'd rather have it in the sdk and explicitly import it (even if that means that we kinda define the methods in every plugin) than using some global vars. 
For me this is not an architectural decision as it form a design point view doesn't matter. It's rather a trade of between the need to maintain more than one repository and ease of use. I would vote for ease of use, as it should be dead simple to build a plugin! But I'm happy to refactor it, if the rest of the team doesn't sees value in having these as part of the SDK. 

/cc @pierlo-upitup @philipnrmn @Poltergeist